doc/rest-api.yaml

```diff
@@ -1,4 +1,41 @@
 definitions:
```

Comment:
Another nit, normal commit message for this is `doc/rest-api:Refresh swagger YAML`